src/test/java/com/puppycrawl/tools/checkstyle/gui/ParseTreeTablePModelTest.java

```diff
@@ -163,8 +165,103 @@ public void testJavadocChild() {
         Assert.assertTrue(javadoc instanceof DetailNode);
         Assert.assertEquals(JavadocTokenTypes.JAVADOC, ((DetailNode) javadoc).getType());
         final Object javadocChild = parseTree.getChild(javadoc, 2);
-        Assert.assertTrue(javadoc instanceof DetailNode);
```

Comment:
This is a redundant assertion, given we already do it in line 165. Unless this is meant to be javadocChild?